mita/openstack.py

```diff
@@ -135,17 +135,29 @@ def get_driver_v2(self):
         """
         return self.driver_v2 if self.driver_v2 else self.get_driver(api_version="2.2")
 
-    def _get_image(self) -> NodeImage:
-        """Return the glance image reference."""
-        try:
-            return [
-                i for i in self.driver_v2.list_images() if i.name == self.image_name
-            ][0]
-        except IndexError:
-            raise ResourceNotFound("Image {} not found".format(self.image_name))
-        except BaseException as be:  # noqa
-            logger.error(be)
-            raise OpenStackDriverError("Encountered an unknown exception.")
+    def _get_image(self, name: Optional[str] = None) -> NodeImage:
+        """
+        Return the NodeImage instance using the image_name or the provided name.
+
+        Args:
+            name: The name of the image.
+
+        Return:
+            NodeImage instance that is referenced by the name.
+
+        Raises:
+            ResourceNotFound - when no resource was found.
+        """
+        name = self.image_name if name is None else name
+        url = f"/v2/images?name={name}"
+        logger.info("Gathering image details: %s", name)
```

Comment:
I think we should pull out these `logger.info` statements, to reduce the overall cephci log size